ci/kind/kind-setup.sh

```diff
@@ -263,6 +263,11 @@ networking:
   kubeProxyMode: $KUBE_PROXY_MODE
 nodes:
 - role: control-plane
+  extraPortMappings:
+    - containerPort: 10349
+      hostPort: 10349
```

Comment:
I guess this is only meaningful if the controller actually runs on the control-plane Node, which is the case when running `hack/make-metrics-doc.sh` since we only provision one Node. Maybe we should skip port mappings altogether if `NUM_WORKERS > 0` (and add a useful comment).